### ISSUE TITLE
Apt repository does not work properly

### ISSUE THREAD

**Issue by dazebug** [sbt/sbt#2928]
## steps

```
echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
sudo apt-get update
sudo apt-get install sbt
```

## problem

```
Executing: /tmp/tmp.DepzG443Kn/gpg.1.sh --keyserver
hkp://keyserver.ubuntu.com:80
--recv
2EE0EA64E40A89B84B2DF73499E82A75642AC823
gpg: requesting key 642AC823 from hkp server keyserver.ubuntu.com
gpg: key 642AC823: "sbt build tool <scalasbt@gmail.com>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
Hit:1 http://ap-northeast-2.ec2.archive.ubuntu.com/ubuntu xenial InRelease
Get:2 http://ap-northeast-2.ec2.archive.ubuntu.com/ubuntu xenial-updates InRelease [102 kB]
Get:3 http://ap-northeast-2.ec2.archive.ubuntu.com/ubuntu xenial-backports InRelease [102 kB]
Get:4 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]
Hit:5 http://ppa.launchpad.net/webupd8team/java/ubuntu xenial InRelease
Ign:6 https://dl.bintray.com/sbt/debian-experimental  InRelease
Ign:7 https://dl.bintray.com/sbt/debian-experimental  Release
Ign:8 https://dl.bintray.com/sbt/debian-experimental  Packages
Ign:9 https://dl.bintray.com/sbt/debian-experimental  Translation-en
Ign:8 https://dl.bintray.com/sbt/debian-experimental  Packages
Ign:9 https://dl.bintray.com/sbt/debian-experimental  Translation-en
Ign:8 https://dl.bintray.com/sbt/debian-experimental  Packages
Ign:9 https://dl.bintray.com/sbt/debian-experimental  Translation-en
Ign:8 https://dl.bintray.com/sbt/debian-experimental  Packages
Ign:9 https://dl.bintray.com/sbt/debian-experimental  Translation-en
Ign:8 https://dl.bintray.com/sbt/debian-experimental  Packages
Ign:9 https://dl.bintray.com/sbt/debian-experimental  Translation-en
Err:8 https://dl.bintray.com/sbt/debian-experimental  Packages
  404  Not Found
Ign:9 https://dl.bintray.com/sbt/debian-experimental  Translation-en
Fetched 306 kB in 3s (97.7 kB/s)
Reading package lists... Done
W: The repository 'https://dl.bintray.com/sbt/debian-experimental  Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch https://dl.bintray.com/sbt/debian-experimental/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists... Done
Building dependency tree
Reading state information... Done
Package sbt is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'sbt' has no installation candidate
```

## expectation

Install the sbt w/o errors

## notes

I tested it on ec2 ubuntu instance.
http://www.scala-sbt.org/0.13/docs/Installing-sbt-on-Linux.html

**Comment by eed3si9n** [comment]
Do you know why it is hitting `https://dl.bintray.com/sbt/debian-experimental` (note *experimental*) instead of https://dl.bintray.com/sbt/debian/?

**Comment by ksilin** [comment]
perhaps because -experimental is the recommendation in the current 1.0 docs? 

http://www.scala-sbt.org/1.0/docs/Installing-sbt-on-Linux.html

echo "deb https://dl.bintray.com/sbt/debian-experimental /" | sudo tee -a /etc/apt/sources.list.d/sbt.list

**Comment by dazebug** [comment]
@eed3si9n 
@ksilin is correct. It was recommended way to install it. I resolved it with `debian` instead of `debian-experimental`, but if this is correct, then document should be fixed.

**Comment by eed3si9n** [comment]
The current stable version is sbt 0.13.x - http://www.scala-sbt.org/documentation.html

**Comment by peggyl** [comment]
I encountered the same issue when installing sbt over the weekend. 

@eed3si9n is correct that the stable 0.13.x docs use `debian`, but if you Google "install sbt" (as I did), the first result is for http://www.scala-sbt.org/1.0/docs/Setup.html. This is the new, non-stable version that uses `debian-experimental`.

Furthermore, the 1.0 docs page does not link to the 0.13 docs. Whether you are on 0.13 or 1.0, the page thinks that current version _is_ the latest version and the dropdown effectively only shows 0.12.7 and 0.7.7 (not even linked on http://www.scala-sbt.org/documentation.html anymore). 

### 0.13 
<img width="1000" alt="sbt-0.13-docs" src="https://cloud.githubusercontent.com/assets/2467815/22676328/a34d3470-ec9f-11e6-90e7-a23c47e74ede.png">

### 1.0

<img width="1000" alt="sbt-1.0-docs" src="https://cloud.githubusercontent.com/assets/2467815/22676367/de40d6fe-ec9f-11e6-8487-9bb7305af975.png">

**Comment by eed3si9n** [comment]
Makes sense. I should probably take 1.0 docs down until it's released, or somehow mark it as beta.

**Comment by emaxedon** [comment]
I'm getting a similar issue.

Following the guide to install `sbt` from `http://www.scala-sbt.org/download.html`.

When I run:
```
echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
sudo apt-get update
sudo apt-get install sbt
```

I get the following error:
```
Get:1 http://security.ubuntu.com/ubuntu xenial-security InRelease [102 kB]
Hit:2 http://mirrors.digitalocean.com/ubuntu xenial InRelease                                                
Hit:3 http://mirrors.digitalocean.com/ubuntu xenial-updates InRelease                                        
Hit:4 http://mirrors.digitalocean.com/ubuntu xenial-backports InRelease                                
Ign:5 http://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.4 InRelease                              
Hit:6 http://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.4 Release                                
Hit:7 https://deb.nodesource.com/node_7.x xenial InRelease
Ign:8 https://dl.bintray.com/sbt/debian  InRelease
Ign:9 https://dl.bintray.com/sbt/debian  Release
Ign:10 https://dl.bintray.com/sbt/debian  Packages
Ign:11 https://dl.bintray.com/sbt/debian  Translation-en_US
Hit:12 https://repos.sonar.digitalocean.com/apt main InRelease
Ign:13 https://dl.bintray.com/sbt/debian  Translation-en
Ign:10 https://dl.bintray.com/sbt/debian  Packages
Ign:11 https://dl.bintray.com/sbt/debian  Translation-en_US
Ign:13 https://dl.bintray.com/sbt/debian  Translation-en
Ign:10 https://dl.bintray.com/sbt/debian  Packages
Ign:11 https://dl.bintray.com/sbt/debian  Translation-en_US
Ign:13 https://dl.bintray.com/sbt/debian  Translation-en
Ign:10 https://dl.bintray.com/sbt/debian  Packages
Ign:11 https://dl.bintray.com/sbt/debian  Translation-en_US
Ign:13 https://dl.bintray.com/sbt/debian  Translation-en
Ign:10 https://dl.bintray.com/sbt/debian  Packages
Ign:11 https://dl.bintray.com/sbt/debian  Translation-en_US
Ign:13 https://dl.bintray.com/sbt/debian  Translation-en
Err:10 https://dl.bintray.com/sbt/debian  Packages
  404  Not Found
Ign:11 https://dl.bintray.com/sbt/debian  Translation-en_US
Ign:13 https://dl.bintray.com/sbt/debian  Translation-en
Fetched 102 kB in 2s (47.0 kB/s)
Reading package lists... Done
W: The repository 'https://dl.bintray.com/sbt/debian  Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch https://dl.bintray.com/sbt/debian/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

I'm running the command on `Ubuntu 16.04.2 LTS`. 

Any ideas?

**Comment by dwijnand** [comment]
The debian-experimental / sbt 1.0 doc issue has been solved, see sbt/website#384.
